src/clj/ataru/applications/application_service.clj

```diff
@@ -90,6 +90,14 @@
     {}
     (application-store/get-application-hakukohde-reviews application-key)))
 
+(defn- parse-application-attachment-reviews
+  [application-key]
+  (reduce
+   (fn [acc {:keys [attachment-key state hakukohde]}]
+     (update-in acc [(or hakukohde :form)] assoc attachment-key state))
```

Comment:
Kantaan on käsittääkseni tallennettu `hakukohde` sarakkeeseen `"form"` jos hakemuksella ei ole hakukohteita? Jollon riittäis `(assoc-in acc [hakukohde attachment-key] state)`.

Reply:
:+1: